src/ucs/debug/debug.c

```diff
@@ -1161,7 +1165,12 @@ static int ucs_debug_backtrace_is_excluded(void *address, const char *symbol)
            !strcmp(symbol, "__ucs_log") ||
            !strcmp(symbol, "ucs_debug_send_mail") ||
            (strstr(symbol, "_L_unlock_") == symbol) ||
-           (address == ucs_debug_signal_restorer);
+#if defined(HAVE_SIGACTION_SA_RESTORER)
+           (address == ucs_debug_signal_restorer)
```

Comment:
move || to the beginning of this line instead of end of line 1167, then would not need else
or make this the first line in the condition instead of last